pom.xml

```diff
@@ -66,7 +66,7 @@
                     <archive>
                         <manifest>
                             <addClasspath>true</addClasspath>
-                            <mainClass>sparksoniq.Main</mainClass>
+                            <mainClass>org.mongodb.Main</mainClass>
```

Comment:
I know it's not review ready yet and I may be missing something here but shouldn't this be rumbledb? I didn't understand the mongodb part

Reply:
Absolutely. Good catch!